### DISCUSSION TITLE
A couple of (related) questions

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1199] [General]
Question 1;

If in amiberry.conf you have set `read_config_descriptions=no` and start amiberry, that is so ... however, if you hit the Restart button, when the GUI is restarted, the config description is displayed. Is this a bug?

https://www.youtube.com/watch?v=l7SjQta_LNU

Question 2;

In wiki/GUI:-About & the About panel helptext, we have the following;

>Restart: This button will stop emulation (if running), reload Amiberry and reset the configuration. This has a similar effect as if you Quit >and Start Amiberry again.

I'm just wondering, should that read ..  "reload Amiberry and reset the **currently loaded** configuration"  ....as that appears to be the behavior here?

TIA

**Comment by midwan** [comment]
Regarding the config file descriptions:
The option disables scanning each config file in the list, to read the description it contains, in order to show it in that list. This part works as expected, even if you Restart.
Once you have loaded a config file, the description is part of it, so it's normal to show it in the fields below, as the last loaded config. The List remains the same as you can see (the descriptions did not appear).

The idea behind this was one of performance: if you end up having thousands of config files in there (and yes, some people did that), opening each file to read the description ends up taking a significant amount of time. Disabling that feature speeds up this quite a lot, so the option was added to control that behavior.

**Comment by midwan** [comment]
Regarding Question 2: Yes, that should be rephrased

**Comment by giantclambake** [comment]
Understood, thanks for updating helptext ;)
